tests/testapps/CTCatalog/CTCatalog/Tests/CTCFrameTestViewController.mm

```diff
@@ -37,6 +41,7 @@ - (void)drawRect:(CGRect)rect {
     CGContextScaleCTM(context, 1.0f, -1.0f);
     CGContextSetTextPosition(context, 0.0, 10.0);
     CTLineDraw(_lineRef, context);
+    CFRelease(_lineRef);
```

Comment:
(`drawRect:` can be called multiple times, and does not signify the end of the ownership of `_lineRef`)

---
In reply to: [79957313](https://github.com/Microsoft/WinObjC/pull/1010#discussion_r79957313) [](ancestors = 79957313)